libpod/container.go

```diff
@@ -151,6 +153,8 @@ type ContainerConfig struct {
 	SharedNamespaceMap map[string]string `json:"sharedNamespaces"`
 	// Time container was created
 	CreatedTime time.Time `json:"createdTime"`
+	// HooksDirPath location of oci hooks config files
+	HooksDirPath string `json:"hooks_dir_path"`
```

Comment:
Is this wired into the database? I think it needs to be, so we can support `podman create` -> `podman start` - start will need to know where the hooks live

Reply:
Also, is this something we should set per-container? I feel like we should be doing this for the entire runtime, not per-container.

Reply:
It can be specified per container, because of the way podman works. But ordinarily it should be for the entire runtime.